isis/src/mro/apps/hideal2pds/main.cpp

```diff
@@ -197,6 +197,11 @@ void IsisMain() {
   image.addKeyword(PvlKeyword("SAMPLE_BIT_MASK", toString((int)pow(2.0, (double)nbits) - 1)),
                    Pvl::Replace);
 
+  // SAMPLE_BITS defaults to 16 but should be 8 when BITS=8
```

Comment:
Changing the mask without changing the bit type of the output doesn't make sense to me. See #4006. These two keywords have to be in sync with each other. See line 131 in this file. It appears to be setting unsignedword for an 8bit output request. I wonder if the mask would work correctly without the fix below if line 131 was set to unsignedbyte????

Reply:
Good eye. Changing that output type to unsignedbyte produces correct mask and samples_bits values in the output pvl for BITS=8.

Line 197 is still needed for BITS values 9-15, because the mask always defaults to the full 16 bits. ( #3978 )

Also, I was under the impression that (output pvl) SAMPLE_BITS should be 16 for (user-entered) BITS=[9, 16] and 8 for BITS=8. Is that correct?